Assets/Scripts/PlayerSelect/PlayerEyeManager.cs

```diff
@@ -0,0 +1,86 @@
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+public class PlayerEyeManager : MonoBehaviour
+{
+
+    private Material eyeMaterial;
+
+    public void Start()
+    {
+        // Find eye material
+        SkinnedMeshRenderer eyeRenderer = transform.GetChild(0).GetComponent<SkinnedMeshRenderer>();
+        eyeMaterial = eyeRenderer.materials[1];
+        resetEyePosition();
+    }
+
+    /// <summary>
+    /// Resets eye position to default. Looking straight ahead.
+    /// </summary>
+    public void resetEyePosition()
+    {
+        eyeMaterial.SetFloat("_LeftUpperLid", 0.12f);
+        eyeMaterial.SetFloat("_LeftLowerLid", 0.2f);
+        eyeMaterial.SetFloat("_RightUpperLid", 0.12f);
+        eyeMaterial.SetFloat("_RightLowerLid", 0.2f);
+
+        eyeMaterial.SetVector("_LeftEye", new Vector4(0, -0.5f));
+        eyeMaterial.SetVector("_RightEye", new Vector4(0, -0.5f));
+
+        eyeMaterial.SetFloat("_Angery", 0.2f);
+    }
+
+    /// <summary>
+    /// Moves the eyes to the players left.
+    /// </summary>
+    /// <param name="targetPosition"></param>
+    public void lookLeft(float targetPosition)
+    {
+        eyeMaterial.SetVector("_RightEye", new Vector4(targetPosition, 0.0f));
+        eyeMaterial.SetVector("_LeftEye", new Vector4(targetPosition, 0.0f));
```

Comment:
I just wanna see some interpolation here, but we can smooth these later

Reply:
Was briefly discussed during a meeting on Wednesday 31.01. and deemed to not be an immediate necessity. We could definitely (and should probably) fix it later though